src/AddressUkPafLookup.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+
+namespace Gcd\UkAddressLookup;
+
+
+use Rhubarb\Leaf\Presenters\Controls\CompositeControlPresenter;
+
+class AddressUkPafLookup extends CompositeControlPresenter
+{
+    private $defaultValues;
+    protected $view;
+    const pafServerUrl = "http://paf.gcdtech.com/paf-data.php?simple=1&api=2&output=json";
+
+    public function __construct($name = "")
+    {
+        parent::__construct($name);
+    }
+
+    protected function initialiseModel()
+    {
+        parent::initialiseModel();
+
+        $this->model->Country = "GB";
+    }
+
+    protected function createView()
+    {
+        $view = new AddressUkPafLookupView();
+        return $view;
+    }
+
+    protected function configureView()
+    {
+        parent::configureView();
+        $this->view->defaultValues = $this->defaultValues;
```

Comment:
What is defaultValues for? It doesn't seem to be set anywhere or used by the view.